src/java/htsjdk/samtools/util/Locatable.java

```diff
@@ -22,4 +22,63 @@
      * @return 1-based closed-ended position, undefined if getContig() == null
      */
     int getEnd();
+
+
+    /**
+    * @return the 0-based start position (from the GA4GH spec).
+    */
+    default long getGA4GHStart() {return this.getStart() - 1; }
```

Comment:
These are a bit weird.  We found a number of places in gatk where we needed to interface with 0-based systems, and having these was convenient.  I think it's better to have them in here than having people do the conversion by hand, but we can keep them as methods in GATK.  If you think they're more clutter than useful I don't mind leaving them in GATK.